lib/jekyll-redirect-from/redirector.rb

```diff
@@ -79,8 +79,10 @@ def config_github_url(site)
       end
     end
 
-    def config_baseurl(site)
-      site.config.fetch('baseurl', nil)
+    def config_url(site)
+      url = site.config.fetch('url', nil) || ""
+      baseurl = site.config.fetch('baseurl', nil) || ""
+      File.join url, baseurl
```

Comment:
`URI.join` seems to fail for relative urls. Thus, I went with `File.join` like [a few lines above](https://github.com/jekyll/jekyll-redirect-from/blob/master/lib/jekyll-redirect-from/redirector.rb#L68).

See discussion of my first commit.